website/project/views/contributor.py

```diff
@@ -560,6 +564,11 @@ def notify_added_contributor(node, contributor, auth=None, throttle=None, email_
     elif not contributor.is_registered:
         unreg_contributor_added.send(node, contributor=contributor, auth=auth, email_template=email_template)
 
+def get_preprint(node):
+    try:
+        return PreprintService.objects.get(node=node)
```

Comment:
A single node can have more than one preprint associated with it, which means `get` could cause some problems. 

That's why, for example, node loops over every preprint it is associated with on save ([see here](https://github.com/CenterForOpenScience/osf.io/blob/develop/osf/models/node.py#L1533))